assets/js/components/ChargingPlansSettings.vue

```diff
@@ -87,12 +88,22 @@ export default {
 	},
 	emits: ["static-plan-removed", "static-plan-updated", "repeating-plans-updated"],
 	data: function () {
+		// If this.plans exists the static plan is active
+		// However no such key exists on the plan-pbject, so we have to add it
+		let staticPlan = {};
+		if (this.plans.length > 0) {
+			staticPlan = {
+				...this.plans[0],
+				active: true,
+			};
+		}
 		return {
 			tariff: {},
 			plan: {},
 			activeTab: "time",
 			isPreview: false,
 			debounceTimer: null,
+			previewPlans: { repeating: this.repeatingPlans, static: staticPlan },
```

Comment:
I'm not convinced about this rather complex structure. We could also decide to only implement the previewing feature for the static plan. Right now It's not really clear to me what the preview would show if you have multiple plans. And additionally I'm not 100% clear what a user would expect the preview to show. We could also decide to remove the previewing feature all together or add a manuell element (radio-button like) to each plan (maybe an eye icon 👁️) to let the user choose what he wants to preview. But right now, I'd propose keeping it simple and starting with limiting it to the static plan.